js/widgets/forms/clearButton.js

```diff
@@ -13,6 +13,11 @@ define( [
 
 	$.widget( "mobile.textinput", $.mobile.textinput, {
 		options: {
+			classes: {
+				"ui-textinput-clear-button":
+					"ui-button ui-button-icon-only ui-corner-all ui-button-right",
```

Comment:
`ui-button` is not a theme class its a structure class from another widget `ui-widget` classes from another widget should never be part of classes. Use the "extra" param of `_add/_remove/_toggleClass` to add these. this should just be `ui-corner-all`